### DISCUSSION TITLE
A small question about "ignore tiling when there's only 1 tile or nothing to do :)"

### DISCUSSION THREAD

**Discussion by Aernil-S** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#256] [Q&A]
I don't know if it is an issue or just I did someting wrong. But when I enabled the Tiled Diffusion in txt2img, it just keeps printing "ignore tiling when there's only 1 tile or nothing to do :)" and just seems noting happend. The picture however still shows up in one whole piece.
Already read the thread #186 but I think we got different problem. That thread doesn't helps much.
![2023-06-22 110850](https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111/assets/110728868/320ad3d8-8557-467c-9450-0480d56766bc)

**Comment by RoscoRiguez** [comment]
I dont know if you ever found a solution, however you need to decrease the Latent Tile Width/Height

Just try lowering each slider down and generate until it kicks in.

**Comment by pkuliyi2015** [comment]
Recommended tile size for both decoder and encoder is 64-128. Please avoid sizes out of this range.
